src/components/LogPane/index.tsx

```diff
@@ -25,7 +25,7 @@ const LogPane: React.FC<LogPaneProps> = ({
     cache.clearAll();
     listRef.current?.recomputeRowHeights();
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [wrap, filters]);
+  }, [wrap, `${filters}`]);
```

Comment:
This hook was being called too much because every `filters` array was being interpreted as a new reference :\

A way to get around this is to stringify the array so that we'll run the hook only when the string changes